packages/grpc-js/gulpfile.ts

```diff
@@ -81,6 +81,21 @@ const runTests = checkTask(() => {
   );
 });
 
-const test = gulp.series(install, copyTestFixtures, runTests);
+const testWithAlternativeResolver = checkTask(() => {
+  process.env.GRPC_NODE_USE_ALTERNATIVE_RESOLVER = 'true';
```

Comment:
Unfortunately, I think this is impacting other test jobs. I think this alternative will work: create a new file called something like `alternate-resolver.env` that just says `GRPC_NODE_USE_ALTERNATIVE_RESOLVER=true` and add an option in the `mocha` command a couple of lines down that says `nodeOption: 'env-file=alternate-resolver.env'`.

Reply:
If you are referring to this, I don't see a `nodeOption` or something similar from the documentation. 

```
mocha({
    reporter: 'mocha-jenkins-reporter',
    require: ['ts-node/register'],
  })
```

Reply:
That is what I'm talking about. The gulp-mocha documentation says

> Options are passed directly to the mocha binary, so you can use any its [command-line options](http://mochajs.org/#usage) in a camelCased form.

The mocha command line options documentation includes this line:

```
 -n, --node-option  Node or V8 option (no leading "--")                 [array]
```

Reply:
the node `--env-file` was added only on node 20 though right? will it work with the CI running on 14 and 16?

Reply:
You're right, that probably won't work. So, here's another alternative: add another target that just sets `process.env.GRPC_NODE_USE_ALTERNATIVE_RESOLVER = 'false';` and add it to the end of the `test` series.

Reply:
Unfortunately, I don't think this quite works because at the top level, the test suites are run in parallel. I think changing [this line](https://github.com/grpc/grpc-node/blob/master/gulpfile.ts#L44) to run in series should fix it.